src/components/userCommentBox.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  TextField,
+  DialogActions,
+  Button,
+  CircularProgress,
+  Paper,
+} from '@material-ui/core'
+import Rating from 'material-ui-rating'
+
+const UserCommentBox = ({
```

Comment:
What is the difference to UserCommentDialog?

Reply:
the userbox is not a dialog and removes one step to comments which is clicking the button to comment, should be a simple box where you input your comment, same is done on all platforms for a better UX